litestar/response/file.py

```diff
@@ -370,7 +367,6 @@ def to_asgi_response(
         """Create an :class:`ASGIFileResponse <litestar.response.file.ASGIFileResponse>` instance.
 
         Args:
-            app: The :class:`Litestar <.app.Litestar>` application instance.
```

Comment:
is this related ?

Reply:
No, I just noticed this was left in there. The param doesn't exist anymore 🤷